### DISCUSSION TITLE
Request information

### DISCUSSION THREAD

**Discussion by caveDweller22** [microsoft/MixedRealityToolkit-Unity#9467] [Q&A] [answered]
I would like to use the Windows Mixed Reality toolkit (MRTK) to set up a unity immersive project that will allow me to create an executable that I can distribute to users in my company. Our company has restricted access to the Microsoft Store. Can an app be produced from unity that you can distribute to other users without the use of Microsoft Store? (via copying an EXE to their machine)? Can you direct me to information that may answer this question and/or Describe the process step by step of completing this task?

**Comment by elbuhofantasma** [comment, marked as answer]
Hello @caveDweller22 

Our official documentation for deploying to HoloLens is [located here.](https://docs.microsoft.com/en-us/windows/mixed-reality/develop/unity/tutorials/mr-learning-base-02#building-your-application-to-your-hololens-2)

If you would like to build from the command line, you can take a look at [Unity's document](https://docs.unity3d.com/Manual/CommandLineArguments.html) on it.

**Reply by david-c-kline** [reply]
@caveDweller22, if your company users have Steam installed, you can build a Standalone application targeting OpenVR and it will run on Windows Mixed Reality Immersive headsets using the following instructions

https://docs.microsoft.com/en-us/windows/mixed-reality/enthusiast-guide/using-steamvr-with-windows-mixed-reality#:~:text=%20Using%20SteamVR%20with%20Windows%20Mixed%20Reality%20,and%20turn%20on%20your%20motion%20controllers.%20More

You can also build as a Universal Windows Platform (UWP) app and sideload it using https://docs.microsoft.com/en-us/windows/application-management/sideload-apps-in-windows-10
